sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -217,12 +216,17 @@ internal class RadarLocationManager(
                     this.startLocationUpdates(options.desiredAccuracy, options.desiredMovingUpdateInterval, options.fastestMovingUpdateInterval)
                 }
 
-                if (options.useMovingGeofence && location != null) {
+                if (options.useMovingGeofence) {
```

Comment:
We should make this consistent with how we do it for stopped geofences above, which is also [how ios does it](https://github.com/radarlabs/radar-sdk-ios/blob/681c62c9c58daee60896d3b07ed4dbe2ca11f9b1/RadarSDK/RadarLocationManager.m#L367-L373))

Looks like ios and android also differ in the null check in `replaceBubbleGeofence`.

Reply:
https://github.com/radarlabs/radar-sdk-android/pull/349/commits/ce4aec0555ccd812a26a8af6de31bdf43ab4997a